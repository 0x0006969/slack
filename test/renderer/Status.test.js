const { Status } = require('./../../lib/slack/renderer/Status');
const combinedStatus = require('./../fixtures/combined_status.json');

describe('Status rendering', () => {
  test('works', async () => {
    const singleStatus = combinedStatus.statuses[1];
    const statusAttachment = new Status(
      singleStatus,
    );
    const rendered = statusAttachment.renderAttachment();
    expect(rendered).toMatchSnapshot();
  });
});
