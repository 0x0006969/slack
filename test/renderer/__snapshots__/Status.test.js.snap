// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status rendering works 1`] = `
Object {
  "author_icon": "https://avatars2.githubusercontent.com/oa/1508?v=4",
  "author_link": "https://travis-ci.org/github-slack/app/builds/278661144?utm_source=github_status&utm_medium=notification",
  "author_name": "continuous-integration/travis-ci/push",
  "color": "#28a745",
  "fallback": "The Travis CI build passed",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "The Travis CI build passed",
}
`;
